toolsets/oracle-operator/oracle-operator.toml

```diff
@@ -0,0 +1,12 @@
+# configurations related to runner
```

Comment:
Maybe add a paragraph in README to show what fields/parameters/etc can be used in this config file?

Reply:
Sure. Will do.